Packages/com.unity.inputsystem/InputSystem/Utilities/ReadOnlyArray.cs

```diff
@@ -81,6 +81,11 @@ public int IndexOf(Predicate<TValue> predicate)
             return -1;
         }
 
+        public int IndexOf(TValue value)
```

Comment:
Is this method actually used anywhere? Left in on purpose?

Reply:
You're right, it's not needed anymore. I removed it.